db/migrate/20220103063359_create_articles.rb

```diff
@@ -0,0 +1,8 @@
+class CreateArticles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :articles do |t|
+      t.string :title
+      t.text :description
```

Comment:
Whats the difference between `string` & `text` data type?

Reply:
`string` is relatively shorter than `text` (according to google, the `string` has a limit up to 255 characters while `text` is 30,000 characters. String is suitable for things that don't require long text like title, name, etc while text is suitable for long text like descriptions, paragraphs etc.

Reply:
in rdbms terms, string == varchar

take note